src/runtime/runtime_internal.h

```diff
@@ -207,9 +209,23 @@ typedef SinkPrinter debug;
 extern WEAK void halide_use_jit_module();
 extern WEAK void halide_release_jit_module();
 
+template <typename T>
```

Comment:
runtime internal functions with inline linkage tend to break on some platforms. Mark these as __attribute__((alwaysinline))